src/js/index.js.rst

```diff
@@ -228,6 +230,78 @@ Otherwise, the line is regular text with optional indentation.
     const parser = peg.generate(rules.join(''));
 
 
+Source Maps
+-----------
+Calculate `source maps`_ for Literacy output. The ``.js.rst`` file format is
+impractical without access to debugger tools or stacktrace line number
+translation.
+
+.. _source map: https://www.html5rocks.com/en/tutorials/developertools/sourcemaps/
```

Comment:
`maps`/`map` anchor doesn't seem to match.

Reply:
Done.